spacy/tokens/doc.pyx

```diff
@@ -1915,6 +1999,70 @@ cdef int [:,:] _get_lca_matrix(Doc doc, int start, int end):
     return lca_matrix
 
 
+cdef const unsigned char[:] _get_utf16_memoryview(str unicode_string, const bint check_2_bytes):
+    """
+    Return a memory view of the UTF-16 representation of a string with the default endianness of the platform.
+    Throw a ValueError if *check_2_bytes == True* and one or more characters in the UTF-16 representation
+    occupies four bytes rather than two.
+    """
+    cdef const unsigned char[:] view = unicode_string.encode("UTF-16")
+    view = view[2:] # first two bytes express endianness
```

Comment:
We can ignore endianness, because each two-byte pair is passed with the default endianness of whatever platform we are on to murmurhash, which takes endianness into account. The tests confirm that the same hash values are returned on all three platforms.